bottlerocket/agents/Cargo.toml

```
@@ -26,6 +26,7 @@ k8s-openapi = { version = "0.17", default-features = false, features = ["v1_24"]
 kube = { version = "0.78", default-features = false, features = ["config", "derive", "client"] }
 log = "0.4"
 maplit = "1"
+openssh = {version = "0.9.9", features = ["native-mux"]}
```

nit:

Also does this introduce a requirement in the SDK?

```suggestion
openssh = { version = "0.9", features = ["native-mux"] }
```

---

Not that I'm aware of.

---

+1 to the version adjustment.

---

Missed it last time. Should be all set now.